requirements.md

```diff
@@ -4,7 +4,7 @@ This document outlines the [input requirements](#inputs) for files to be submitt
 
 ## Inputs
 
-The DTV pipeline is triggered when a single `.tar` file is uploaded to the designated AWS S3 bucket. This `.tar` file should contain the file(s) to be transformed, any supplementary distribution files, and a `manifest.json` file for configuring the pipeline.
+The DTV pipeline is triggered when a single or multiple `.tar` file/files is uploaded to the designated AWS S3 bucket. This `.tar` file should contain the file(s) to be transformed, any supplementary distribution files, and a `manifest.json` file for configuring the pipeline.
```

Comment:
Change to:
"The DTV pipeline is triggered when one or more `.tar` files is uploaded to the designated AWS S3 bucket. Each `.tar` file should contain the file(s) to be transformed, any supplementary distribution files, and a `manifest.json` file for configuring the pipeline.